internal/util/etcd/etcd_util.go

```diff
@@ -34,14 +33,21 @@ var (
 )
 
 // GetEtcdClient returns etcd client
-func GetEtcdClient(cfg *paramtable.EtcdConfig) (*clientv3.Client, error) {
```

Comment:
Why not use EtcdConfig as a parameter that will reduce a lot of redundant code?

Reply:
cannot do that, it will cause cycle dependency